### DISCUSSION TITLE
How to send double quote in tui command

### DISCUSSION THREAD

**Discussion by jharming** [ansys/pyfluent#1116] [Q&A] [answered]
I try to send this TUI command
tui.define.boundary_conditions.wall("inlet", "0", "no", "0", "yes", "copper", "no", "no", "0", "yes", "no", '"source_in"', "no", "0", "no", "0", "no", "1")
But I get the following
"Error": name 'source_in' is not defined
But if I do a list of named expression I do have it listed. I fluent standalone I simply send the tui command with double quote. But this does not work in python. I can't find in documentation what to do.

**Comment by sravanansys** [comment, marked as answer]
@jharming This is working fine for me in setting a named expression for wall temperature through TUI and settings API. I have a named expression 'temp' defined as 300[K].

![image](https://user-images.githubusercontent.com/99287292/205819705-485328b5-d466-4d4d-a55c-de5212d5f497.png)

**Reply by jharming** [reply]
@snallamoansys  thanks for the reply. I see now it works both solutions. I like the first one the best since it is easier to read what is going on.
